public_html/wp-content/mu-plugins/blocks/assets/src/shared/featured-image/inspector-control.js

```diff
@@ -2,48 +2,33 @@
  * WordPress dependencies.
  */
 const { Component } = wp.element;
-const { PanelBody, PanelRow, TextControl, BaseControl, SelectControl } = wp.components;
+const { PanelBody, PanelRow, TextControl, BaseControl, SelectControl, ToggleControl } = wp.components;
 const { __ } = wp.i18n;
+import './style.scss';
 
 /**
  * Not sure if these sizes are actually useful.
  */
 const sizePresets = [
 	{
 		label : __( 'Thumbnail', 'wordcamporg' ),
-		value : '150x150',
+		value : '150',
 	},
 	{
-		label : __( 'Small - 1:1', 'wordcamporg' ),
-		value : '200x200',
+		label : __( 'Thumbnail - Large', 'wordcamporg' ),
+		value : '926',
 	},
 	{
-		label : __( 'Small - 3:2', 'wordcamporg' ),
-		value : '270x180',
+		label : __( 'Medium', 'wordcamporg' ),
+		value : '300',
 	},
 	{
-		label : __( 'Small - 4:3', 'wordcamporg' ),
-		value : '300x225',
+		label : __( 'Medium - Large', 'wordcamporg' ),
+		value : '768',
 	},
 	{
-		label : __( 'Medium - 1:1', 'wordcamporg' ),
-		value : '400x400',
-	},
-	{
-		label : __( 'Medium - 3:2', 'wordcamporg' ),
-		value : '480x320',
-	},
-	{
-		label : __( 'Medium - 4:3', 'wordcamporg' ),
-		value : '480x360',
-	},
-	{
-		label : __( 'Medium - 16:9', 'wordcamporg' ),
-		value : '720x405',
-	},
-	{
-		label : __( 'Large - 16:9', 'wordcamporg' ),
-		value : '1280x720'
+		label : __( 'Large', 'wordcamporg' ),
+		value : '1024',
```

Comment:
Should we grab these dynamically from Core rather than hardcoding them?

Reply:
> Should we grab these dynamically

That was my suggestion above. Is there a blocker to doing this, or some other reason it's still statically defined?

Reply:
I was not able to find a good place in front-end where I can fetch these from. Image block in core seems to generate these from the image itself (which can have different sizes per image). So we cannot use any of the image data to render options for all images.

Reply:
Tracking this in https://github.com/WordPress/wordcamp.org/issues/36. Resolving this conversation.